### ISSUE TITLE
Wrapper for interactive sessions with X11 forwarding

### ISSUE THREAD

**Issue by chnixi** [pitt-crc/wrappers#166]
We have removed the X-forwarding option from crc-interactive.py. It would be good to have an alternative wrapper that launches interactive sessions with X11-forwarding so users can run GUI applications, as described here: https://crc.pitt.edu/resources/htc-cluster/interactive-jobs

**Comment by djperrefort** [comment]
We had problems with the original X11 implementation because the wrappers tried to enable it automatically. A potential work around would be to add an `x11` flag to `crc-interactive` to manually enable x11 forwarding. Would this work for you?

I'm hesitant to add an entirely new application for such a small feature.

**Comment by chnixi** [comment]
Yes, that would work well. I forgot what the original issue with X11-forwarding in crc-interactive was so thought it was safer to create a new app, but we can just extend our existing wrapper.
